### ISSUE TITLE
Paste too long

### ISSUE THREAD

**Issue by Nikos410** [interaapps/pastefy#11]
Hey there!

I've tried creating a somewhat long paste, which always leads to this exception in the log:
```
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.javawebstack.orm.Model.save(Model.java:126)
	at de.interaapps.pastefy.controller.FormDataPasteController.createPaste(FormDataPasteController.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.javawebstack.httpserver.router.RouteBinder$BindMapper.invoke(RouteBinder.java:254)
	at org.javawebstack.httpserver.router.RouteBinder$BindHandler.handle(RouteBinder.java:300)
	at org.javawebstack.httpserver.HTTPServer.execute(HTTPServer.java:338)
	at org.javawebstack.httpserver.HTTPServer$HttpHandler.handle(HTTPServer.java:414)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:500)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:547)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:270)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:117)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.javawebstack.orm.exception.ORMQueryException: Data truncation: Data too long for column 'content' at row 1
	at org.javawebstack.orm.Repo.executeCreate(Repo.java:142)
	at org.javawebstack.orm.Repo.create(Repo.java:106)
	at org.javawebstack.orm.Repo.save(Repo.java:99)
	... 26 more
java.lang.RuntimeException: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.javawebstack.httpserver.router.RouteBinder$BindMapper.invoke(RouteBinder.java:256)
	at org.javawebstack.httpserver.router.RouteBinder$BindHandler.handle(RouteBinder.java:300)
	at org.javawebstack.httpserver.HTTPServer.execute(HTTPServer.java:338)
	at org.javawebstack.httpserver.HTTPServer$HttpHandler.handle(HTTPServer.java:414)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:500)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:547)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:270)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:117)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.javawebstack.orm.Model.save(Model.java:129)
	at de.interaapps.pastefy.controller.FormDataPasteController.createPaste(FormDataPasteController.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.javawebstack.httpserver.router.RouteBinder$BindMapper.invoke(RouteBinder.java:254)
	... 15 more
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.javawebstack.orm.Model.save(Model.java:126)
	... 21 more
Caused by: org.javawebstack.orm.exception.ORMQueryException: Data truncation: Data too long for column 'content' at row 1
	at org.javawebstack.orm.Repo.executeCreate(Repo.java:142)
	at org.javawebstack.orm.Repo.create(Repo.java:106)
	at org.javawebstack.orm.Repo.save(Repo.java:99)
	... 26 more
com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Unterminated string at line 1 column 16001 path $.content
	at com.google.gson.Gson.fromJson(Gson.java:947)
	at com.google.gson.Gson.fromJson(Gson.java:897)
	at com.google.gson.Gson.fromJson(Gson.java:846)
	at com.google.gson.Gson.fromJson(Gson.java:817)
	at org.javawebstack.abstractdata.AbstractElement.fromJson(AbstractElement.java:178)
	at org.javawebstack.httpserver.Exchange.body(Exchange.java:75)
	at org.javawebstack.httpserver.router.RouteBinder$BindMapper.invoke(RouteBinder.java:229)
	at org.javawebstack.httpserver.router.RouteBinder$BindHandler.handle(RouteBinder.java:300)
	at org.javawebstack.httpserver.HTTPServer.execute(HTTPServer.java:338)
	at org.javawebstack.httpserver.HTTPServer$HttpHandler.handle(HTTPServer.java:414)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:500)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:547)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:270)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:117)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:388)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.gson.stream.MalformedJsonException: Unterminated string at line 1 column 16001 path $.content
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1564)
	at com.google.gson.stream.JsonReader.nextQuotedValue(JsonReader.java:1032)
	at com.google.gson.stream.JsonReader.nextString(JsonReader.java:816)
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:702)
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:723)
	at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:698)
	at com.google.gson.internal.bind.TypeAdapters$35$1.read(TypeAdapters.java:894)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	... 26 more
```

You can actually reproduce this by creating a paste containing the above stacktrace as its content.

Is there a way to configure the maximum paste size?

I'm running pastefy using the docker image with the tag `latest`

**Comment by JulianFun123** [comment]
Hey, thanks for letting me know. I fixed the bug in the latest commit and released 6.3.3

**Comment by JulianFun123** [comment]
Hey, thanks for letting me know. I fixed the bug in the latest commit and released 6.3.3. pastefy.ga itself will still have a maximum upload size, but if you're self-hosting it should be fine.
